003_app_L24.py

```
@@ -0,0 +1,164 @@
+            # {/fact}
+            return tel_array
+    except FileNotFoundError:
+        print('numbers.txt Not Found!')
+        sys.exit()
+        
+
+# Get SMS Body from txt file
+# {fact rule=resource-leak@v1.0 defects=1}
+def getSMSBody():
+    try:
+        with open('message.txt', 'r') as myfile:
```

Ruff (0.14.5)

Local variable `myfile` is assigned to but never used

Remove assignment to unused variable `myfile`

(F841)